indykite/resource_authorization_policy.go

```diff
@@ -60,11 +58,25 @@ func resourceAuthorizationPolicy() *schema.Resource {
 				Required:         true,
 				DiffSuppressFunc: structure.SuppressJsonDiff,
 				ValidateFunc: validation.All(
+					validation.StringIsNotEmpty,
 					validation.StringIsJSON,
-					authorizationPolicyValidateJSON,
 				),
 				Description: "Configuration of Authorization Policy in JSON format, the same one exported by Console.",
 			},
+			statusKey: {
+				Type:         schema.TypeString,
```

Comment:
That is not entirely true. Yes we have a boolean to identify active polices in db. But the status is stored as enum in the config node. In the future we may introduce other statuses, e.g: draft, so we need to be able to set specific status

Reply:
OK, so the question is what to do here. Because I don't like numbers like it is now. So either:

1. For now, use boolean as there is just active and inactive. And change later.
2. Change now, and maybe use text like `active`, and `inactive`.

But not the numbers.

Reply:
I will change for the text active, inactive, since it has the benefit of being self descriptive but we can discuss that

Reply:
Yes, that might be better. See `indykite/utilities.go` and `OAuth2GrantTypes` for example, how it is used. Together with `ReverseProtoEnumMap` and `getMapStringKeys`